openfe_gromacs/tests/protocols/test_gromacs_md.py

```diff
@@ -198,6 +196,45 @@ def test_dry_many_molecules_solvent(
         )
 
 
+def test_gather(benzene_system, tmp_path_factory):
```

Comment:
Is this running a full simulation? If so, I would probably add a couple of comments below saying this (replacing the mock.patch stuff) it's worth saying "this is running a full simulation, but it's cheap so it doesn't matter" or something like that (and what the aim of the test is).